src/odb/include/odb/db.h

```diff
@@ -3561,9 +3561,14 @@ class dbITerm : public dbObject
 
   ///
   /// Disconnect this iterm from the net it is connected to.
+  /// Will remove from both mod net and dbNet.
   ///
   void disconnect();
 
+  ///
+  /// Selectively disconnect db_net/mod_net
+  ///
+  void disconnect(bool db_net, bool mod_net);
```

Comment:
I'm not fond of this API as it leaves the db in an inconsistent state.  I would rather both were connected/disconnected together.

Reply:
removed